### PR TITLE
Callback manage documentation

### PR DESCRIPTION
**Pull request by OSoloviova**

### CODE REVIEW

File: docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
+===========
+Callbacks
```
Comment (kolapse2) [inline docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst:2]: please make "=" signs to be in the line with title, it's necessary to fit sphynx markup

File: docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+===========
+
+Envelope callback is functionality which allows to receive notification to respective URL in case of envelope appearance in mailbox and according to configured filters. You can set up callbacks for mailbox in order to receive envelope notifications outside of our platform. This is done via API. 
+How to configure callbacks you can see here - :ref:`Envelope callbacks <envelopeCallback>`
+The administrative panel allows you to view and partially manage callbacks.
+
+Manage callbacks
```
Comment (kolapse2) [inline docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst:9]: Same as for title

File: docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+- authentication type (2) - Basic, None
+- success code (3)
+- filter (4) - list of specified filters for a specific callback
+- actions buttons (5) - settings and delete
+
+.. image:: pic_users/callbacksColumns.png
+   :width: 600
+   :align: center
+
+Check callbacks
```
Comment (kolapse2) [inline docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst:37]: same as for title

File: docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+.. note:: you can send callback again, follow back to previous step (enter envelope UUID) or cancel action.
+
+5. In case if the specified envelope UUID doesn't match the configured filters a message listing mismatches will be displayed. After that you can just follow back to previous step and enter another envelope UUID or cancel action.
+
+.. image:: pic_users/callbacksFilterNotMatch.png
+   :width: 600
+   :align: center  
+
+Delete callbacks
```
Comment (kolapse2) [inline docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst:75]: Same as for title

**Review by kolapse2 — CHANGES_REQUESTED** [review]
please update markup

File: docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst (hunk 1 above)
Comment (OSoloviova) [reply docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst:2]: done

File: docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst (hunk 2 above)
Comment (OSoloviova) [reply docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst:9]: done

File: docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst (hunk 3 above)
Comment (OSoloviova) [reply docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst:37]: done

File: docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst (hunk 4 above)
Comment (OSoloviova) [reply docs/source/pages/howToWorkWithPlatform/adminApplication/callbacks/callbacks.rst:75]: done

**Review by kolapse2 — APPROVED** [review]
Approved

**Pull request merged**